frontend/components/panels/MKRCheckPanel.vue

```diff
@@ -1,121 +0,0 @@
-<template>
```

Comment:
What is the new name of this panel? Github didn't recognise the move, I also couldn't find it 🙂

Reply:
It was left over from #343. Lukas and I were not sure if we wanted to display two components (MKR and allowance check) in one or two panels. 

That is why we merged both versions to check later. As we decided later on to separate them, this component was never used, but I forgot to delete it.